src/index.jsx

```diff
@@ -24,24 +22,50 @@ let DATA_PACKAGE = DATA_PACKAGE || {}
  * - Also any properties can be passed from the back-end using data-* prefix.
  */
 
-dputils.fetchDataPackageAndData(DATA_PACKAGE_URL).then(dpObj => {
-  DATA_PACKAGE = dpObj.descriptor
+let dataPackage
```

Comment:
Is this under test? I'm wondering if this code should be somewhere a bit separate so you can test it e.g. a function taking a DATA_PACKAGE and/or DATA_PACKAGE_URL as an argument like the DataPackageView)